.github/workflows/rust.yml

```diff
@@ -14,14 +14,56 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        rust:
+          - 1.70.0 # docker rust:1-bookworm
+          - 1.63.0 # debian
+          - 1.61.0 # MSRV
+        features:
+          - ""
+          - default
+
     steps:
       - uses: actions/checkout@v3
-      - uses: dtolnay/rust-toolchain@stable
+      - uses: actions-rs/toolchain@v1
```

Comment:
there are a lot of deprecations using this action, see "Summary" in the pipeline, so I migrated to the dtolnay provided one use: `dtolnay/rust-toolchain@master` and you can specify the toolchain

Reply:
Ah, I didn't check the details on those actions. I just checked how the matrix worked in bdk-cli.